src/main/java/com/urswolfer/gerrit/client/rest/http/groups/GroupApiRestClient.java

```diff
@@ -0,0 +1,253 @@
+     * Set group owner.
+     *
+     * @param owner identifier of new group owner.
+     * @throws RestApiException
+     */
+    public void owner(String owner) throws RestApiException {
+        String restPath = getRequestPath() + "/owner";
+        JsonElement result = gerritRestClient.putRequest(restPath, owner);
+    }
+
+    /** @return group description. */
+    public String description() throws RestApiException {
+        String restPath = getRequestPath() + "/description";
+        return gerritRestClient.getRequest(restPath).getAsString();
+    }
+
+    /**
+     * Set group decsription.
+     *
+     * @param description new description.
+     * @throws RestApiException
+     */
+    public void description(String description) throws RestApiException {
+        String restPath = getRequestPath() + "/description";
+        JsonElement result = gerritRestClient.putRequest(restPath, description);
+    }
+
+    /** @return group options. */
+    public GroupOptionsInfo options() throws RestApiException {
+        throw new NotImplementedException();
+    }
+
+    /**
+     * Set group options.
+     *
+     * @param options new options.
+     * @throws RestApiException
+     */
+    public void options(GroupOptionsInfo options) throws RestApiException {
+        throw new NotImplementedException();
+    }
+
+    /**
+     * List group members, non-recursively.
+     *
+     * @return group members.
+     * @throws RestApiException
+     */
+    public List<AccountInfo> members() throws RestApiException {
+        String restPath = getRequestPath() + "/members";
+        JsonElement result = gerritRestClient.getRequest(restPath);
+        return groupsParser.parseGroupMembers(result);
+    }
+
+    /**
+     * List group members.
+     *
+     * @param recursive whether to recursively included groups.
+     * @return group members.
+     * @throws RestApiException
+     */
+    public List<AccountInfo> members(boolean recursive) throws RestApiException {
+        String restPath = getRequestPath() + "/members/?recursive";
+        JsonElement result = gerritRestClient.getRequest(restPath);
+        return groupsParser.parseGroupMembers(result);
+    }
+
+    /**
+     * Add members to a group.
+     *
+     * @param members list of member identifiers, in any format accepted by
+     *     {@link com.google.gerrit.extensions.api.accounts.Accounts#id(String)}
+     * @throws RestApiException
+     */
+    public void addMembers(String... members) throws RestApiException {
+        String restPath = getRequestPath() + "/members";
+
+        // Create an object which can be used to create the json for:
+        // { members: [ "member1", "member2" ] }
+        Map<String, List<String>> memberMap = new HashMap<String, List<String>>();
+        memberMap.put("members", Arrays.asList(members));
+        String json = gerritRestClient.getGson().toJson(memberMap);
+
+        JsonElement result = gerritRestClient.postRequest(restPath, json);
```

Comment:
Variable result is also not necessary here (and in some other places in this file) - please remove it.